Launcher/MainWindow.xaml.cs

```diff
@@ -235,14 +235,14 @@ public MainWindow()
                     file_list files_to_patch = file_list.none;
                     if (!check_files(ref files_to_patch))
                     {
-                        MessageBox.Show("You are using a version of the toolkit not supported by H2Codez, features added by H2Codez will not be available.\nPlease install the orginal version of the toolkit that was distributed on the DVD, as that's the only version H2Codez can patch.",
+                        MessageBox.Show("You are using a version of the toolkit that is not supported by H2Codez, features added by H2Codez will not be available.\nPlease install the orginal version of the toolkit that was distributed on the DVD, as that's the only version H2Codez can patch.",
                          "Version Error!");
                         return;
                     }
 
                     if (!File.Exists(H2Ek_install_path + "H2Codez.dll") || files_to_patch != file_list.none)
                     {
-                        MessageBoxResult user_answer = MessageBox.Show("Your have not installed H2Codez or your version is outdated.\nDo you want to installed H2Codez?",
+                        MessageBoxResult user_answer = MessageBox.Show("You have not installed H2Codez or your version is outdated.\nDo you want to install H2Codez?",
```

Comment:
```suggestion
                        MessageBoxResult user_answer = MessageBox.Show("You have not installed H2Codez or the installation process is incomplete.\nDo you want to install H2Codez?",
```
Not sure how an outdated version could cause that.